module.exports = () => {
  return async (ctx, next) => {
    try {
      // content here
      await next()
    } catch (error) {
      ctx.throw(error)
    }
  }
}
